test/e2e/pull_test.go

```diff
@@ -35,6 +35,42 @@ var _ = Describe("Podman pull", func() {
 
 	})
 
+	It("podman pull multiple images with tag", func() {
+		session := podmanTest.Podman([]string{"pull", "busybox:latest", "alpine:edge"})
```

Comment:
`alpine:edge` does not exist on our mirror on quay.  Please use `alpine`.  Then you can also remove the `"podman pull multiple images without tag"` test.